### PR TITLE
added new card and fixes the card

### PR DESCRIPTION
**Pull request by grixxxx**
fixed card content

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,6 +114,35 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+    <!-- ________ *Neobit* Contributor card START ________  -->
```
Comment (TenTraicion) [inline index.html:117]:
```suggestion

    <!-- ________ *grixxxx* Contributor card START ________  -->
```

File: index.html [hunk 2]
```diff
@@ -114,6 +114,35 @@ <h2>A project for first-time contributions</h2>
...
+<div class="card">
+          <p class="name">grixxxx</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/grixxxx" target="_blank">grixxxx</a>
+              <i class="fab fa-telegram"></i>
+            <a href="t.me/grixonly" target="_blank">grixxxx</a>
```
Comment (TenTraicion) [inline index.html:124]:
```suggestion
            <a href="https://t.me/grixonly" target="_blank">@grixxxx</a>
```

File: index.html [hunk 3]
```diff
@@ -114,6 +114,35 @@ <h2>A project for first-time contributions</h2>
...
+              <i class="fab fa-linkedin"></i>
+            <a href="https://www.linkedin.com/in/delmer-john-ballesteros-b8b683252/" target="_blank">Delmer John Ballesteros</a>
+          </p>
+          <p class="about">Front-End Developer | Student</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://frontendmasters.com/course" target="_blank" title="First Resource">frontend Developer Roadmap</a>
```
Comment (TenTraicion) [inline index.html:133]: It's not a roadmap but a bunch of courses. Kindly check the url or fix the title. Thanks!
```suggestion
                <a href="https://frontendmasters.com/course" target="_blank" title="First Resource">frontend Developer Roadmap</a>
```

File: index.html [hunk 4]
```diff
@@ -114,6 +114,35 @@ <h2>A project for first-time contributions</h2>
...
+              <li>
+                <a href="https://frontendmasters.com/courses/#free" target="_blank" title="Second Resource">FrontendMasters</a>
+              </li>
+              <li>
+                <a href="https://scrimba.com" target="_blank" title="Third resource">Scrimba</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+ <!-- ________ *Neobit* Contributor card END ________  -->
```
Comment (TenTraicion) [inline index.html:144]:
```suggestion
 <!-- ________ *grixxxx* Contributor card END ________  -->
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@grixxxx, Kindly make the required changes and commit. 

@CandidDeer, @Syknapse or @win21H2 kindly check the css part and give your review. Thanks.

**Comment by grixxxx** [comment]
done

**Comment by CandidDeer** [comment]
Thank you for your contribution and making the requested changes. I have merged your PR now and your card will be live in a moment.

**Pull request merged**